boost-maven/boost-maven-plugin/src/it/test-mpJwt-1.1/pom.xml

```diff
@@ -0,0 +1,274 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Can we rename the IT dir to:  "test-mp-jwt-1.1"... this lets you use patterns like `mvn install -Pit -Dinvoker.test=*mp-jwt*`  with the same naming convention.